packages/pwa/app/pages/account/orders.test.js

```diff
@@ -49,48 +44,11 @@ const MockedComponent = () => {
     )
 }
 
-const server = setupServer(
-    rest.post('*/customers/actions/login', (req, res, ctx) =>
-        res(
-            ctx.delay(0),
-            ctx.set('authorization', `Bearer guesttoken`),
-            ctx.json(mockedRegisteredCustomer)
-        )
-    ),
```

Comment:
I noticed the refactor doesn't include this endpoint, but since tests are passing I'm assuming we didn't use the endpoint here.

Reply:
yes, I don't think we need this anymore.